pkg/node/kube-proxy.go

```diff
@@ -53,21 +52,14 @@ func (s *ProxyOptions) configure(cfg *config.MicroshiftConfig) {
 	if err := s.writeConfig(cfg); err != nil {
 		logrus.Fatalf("Failed to write kube-proxy config: %v", err)
 	}
-	args := []string{
-		"--logtostderr=" + strconv.FormatBool(cfg.LogDir == "" || cfg.LogAlsotostderr),
-		"--alsologtostderr=" + strconv.FormatBool(cfg.LogAlsotostderr),
-		"--v=" + strconv.Itoa(cfg.LogVLevel),
-		"--vmodule=" + cfg.LogVModule,
-	}
+	// Keeping the args in case something must be added in the future
+	args := []string{""}
```

Comment:
```suggestion
	// Keeping the args in case something must be added in the future
	args := []string{}
```

Reply:
I wonder if this makes kubeproxy fail to start, empty arg in the arg list?

Locally we would not see it, because kubeproxy is internal, and already has configured all iptables, etc.